autodistill/helpers.py

```diff
@@ -84,3 +89,41 @@ def split_data(base_dir, split_ratio=0.8):
             "names": names,
         }
         yaml.dump(data, file)
+
+
+def visualize_predictions(
+    file_name: str, detections: sv.Detections, annotation_type: str
+) -> None:
+    if annotation_type not in VALID_ANNOTATION_TYPES:
+        raise ValueError(
+            f"Invalid annotation type. Must be one of {VALID_ANNOTATION_TYPES}"
+        )
+
+    if annotation_type == "box":
+        annotator = sv.BoxAnnotator()
+    elif annotation_type == "mask":
+        annotator = sv.MaskAnnotator()
+
+    image = cv2.imread(file_name)
+
+    image_bgr = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+
+    annotated_image = annotator.annotate(scene=image_bgr, detections=detections)
+
+    sv.plot_image(annotated_image)
+
+
+def split_video_frames(video_path: str, output_dir: str, stride: int) -> None:
+    video_paths = sv.list_files_with_extensions(
+        directory=video_path, extensions=["mov", "mp4"]
+    )
+
+    for name in tqdm(video_paths):
+        image_name_pattern = name + "-{:05d}.png"
```

Comment:
We should do jpg's vs pngs

Reply:
@yeldarby I'm curious why you would pick `jpg` over `png`? Typically you would like to avoid lossy compression when working with vision models.